map/src/drawer/components/tracks/LocalTrackGroup.jsx

```diff
@@ -126,19 +157,28 @@ export default function LocalTrackGroup() {
                         Upload
                     </Button>
                 </label>
+                <Button className={classes.button} variant="contained" component="span" sx={{ml: 2}}
+                        onClick={() => generateLocalClientTracks()}>
+                    Generate
+                </Button>
+            </MenuItem>
+            <MenuItem disableRipple={true}>
+                <Button className={classes.button} variant="contained" component="span" sx={{ml: 3}}
+                        onClick={clearLocalTracks(ctx)}>
+                    Clear
+                </Button>
+                {localGpxFiles.length !== 0 &&
+                    <Tooltip title={
+                        <p>
+                            For saved tracks
```

Comment:
Weird description in tooltip ? is it tooltip for GET Obf (please contact me)

Reply:
`+`